empire.go

```diff
@@ -273,6 +281,9 @@ func (opts SetOpts) Validate(e *Empire) error {
 // Config. If the app has a running release, a new release will be created and
 // run.
 func (e *Empire) Set(ctx context.Context, opts SetOpts) (*Config, error) {
+	ctx, cancel := context.WithTimeout(ctx, TimeoutSet)
```

Comment:
Thinking about this a bit more, these should really be defined inside server/heroku instead of here.